docs/docsource/operation/Orbit_UI/Setting_Up_Orbit/Setting_up_a_Sandbox_Instance.rst

```diff
@@ -17,8 +17,12 @@ your data. To set up a Sandbox instance:
 
    |image0|
 
-#. The Welcome dialog displays:
+#. Click **Register with Google**. You must have a Google ID to authenticate
+   against.
```

Comment:
`authenticate with`?

Reply:
changed to: "You must authenticate using a Google ID."